src/meta/server_state.cpp

```diff
@@ -2344,11 +2344,6 @@ bool server_state::check_all_partitions()
                    ::dsn::enum_to_string(app->status));
             continue;
         }
-        if (app->is_bulk_loading) {
```

Comment:
Does it will produce  error when rebalance the bulk loading app?

Reply:
balance operation is in function `greedy_balancer`, you can see it in pull request #571.